modules/local/download_mgnify_genomes_reference_dbs.nf

```diff
@@ -16,12 +16,16 @@ process DOWNLOAD_MGNIFY_GENOMES_REFERENCE_DBS {
     tuple val(biome), path("bwamem2_index/")                              , emit: bwamem2_index, optional: true
 
     script:
-    def matcher = biome =~ /(.+?)(-v[0-9.\.]+)?$/
+    def matcher = biome =~ /(.+?)(-v[0-9]+-[0-9]+(?:-[0-9]+)?)?$/
```

Comment:
out of curiosity, on what biome previous regexp failed?

Reply:
It failed in any biome because it was looking for the version separated by dot, but the input have '-' instead.

Reply:
We keep it that way because that is the format of the biome names in MGnify website: https://www.ebi.ac.uk/metagenomics/browse/genomes